recipe/activate/activate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#eval `python "$CONDA_PREFIX"/etc/conda/sage-conda-env.py activate`
+
+export SAGE_ROOT="$CONDA_PREFIX"
+export SAGE_LOCAL="$CONDA_PREFIX"
+
+#source "$CONDA_PREFIX/bin/sage-env"
```

Comment:
@jakirkham, @ocefpaf I want to source this script, but I'm having a hard time writing a deactivate script. Have you guys had any experience restoring an environment in a deactivate script?